src/main/java/me/juancarloscp52/entropy/events/db/BouncyBlocksEvent.java

```diff
@@ -0,0 +1,26 @@
+package me.juancarloscp52.entropy.events.db;
+
+import me.juancarloscp52.entropy.Variables;
+import me.juancarloscp52.entropy.events.AbstractTimedEvent;
+import me.juancarloscp52.entropy.events.EventCategory;
+import me.juancarloscp52.entropy.events.EventType;
+
+public class BouncyBlocksEvent extends AbstractTimedEvent {
+    public static final EventType<BouncyBlocksEvent> TYPE = EventType.builder(BouncyBlocksEvent::new).category(EventCategory.MOVEMENT).build();
```

Comment:
To me it doesn't feel like this is a movement-based event. All events that are within this category somehow modify the player input[^1], which this one doesn't do.

[^1]: Apart from the lag event, which I also don't think fits for the same reason, but that's a different story.